azure-keyvault/tests/test_secrets.py

```diff
@@ -80,12 +111,21 @@ def _update_secret(secret):
 
         # update secret with version
         if self.is_live:
-            # wait a second to ensure the secret's update time won't equal its creation time
-            time.sleep(1)
```

Comment:
Did you find 1 second too short a delay?

Reply:
I must have left it that way when I was trying something else.
1 second works fine here.